workflow/rtma3d_pbspro_conus.xml

```diff
@@ -216,22 +223,25 @@
 <!ENTITY PREPOBS_RESERVATION '<nodes>1:ppn=1:vntype=cray_compute:mem=4000MB</nodes><native>-l place=scatter</native><queue>&QUEUE;</queue><account>&ACCOUNT;</account>'>
 
 <!ENTITY PREPFGS_PROC "1">
-<!ENTITY PREPFGS_RESOURCES '<walltime>00:10:00</walltime>'>
-<!ENTITY PREPFGS_RESERVATION '<nodes>1:ppn=1:vntype=cray_compute:mem=4000MB</nodes><native>-l place=scatter</native><queue>&QUEUE;</queue><account>&ACCOUNT;</account>'>
+<!ENTITY PREPFGS_RESOURCES '<walltime>00:15:00</walltime>'>
+<!ENTITY PREPFGS_RESERVATION '<nodes>1:ppn=1:vntype=cray_compute:mem=50GB</nodes><native>-l place=scatter</native><queue>&QUEUE;</queue><account>&ACCOUNT;</account>'>
```

Comment:
Why do we need this increase in memory?

Reply:
@MatthewMorris-NOAA 
Hi Matthew,

I add the step to dump out howv and gust from grib2 to binary (by using wgrib2) and then append the binary data to netcdf firstguess file (by using ncks). the ncks step would generate a temporary 16GB file which requires large memory. So I have to increase the memory usage for this step.

Thank you!

-Gang

Reply:
Sounds good!

Reply:
@MatthewMorris-NOAA 
I tested with mem=20GB for CONUS domain on wcoss2. Code does not crash, but ran with warning message "group memory limit exceeded". So I switch back to 50GB.
Thank you!
-Gang